src/test/ui/panic-runtime/unwind-msvc-no-std-link-to-libcmt.rs

```diff
@@ -0,0 +1,47 @@
+// build-pass
+// compile-flags: -C panic=unwind -C target-feature=+crt-static
+// only-msvc
+// Test that `no_std` with `panic=unwind` under MSVC toolchain
+// doesn't cause error when linking to libcmt.
+
+#![no_std]
+#![no_main]
+#![feature(alloc_error_handler)]
+#![feature(panic_unwind)]
+
+use core::alloc::{GlobalAlloc, Layout};
+
+struct DummyAllocator;
+
+unsafe impl GlobalAlloc for DummyAllocator {
+    unsafe fn alloc(&self, _layout: Layout) -> *mut u8 {
+        core::ptr::null_mut()
+    }
+
+    unsafe fn dealloc(&self, _ptr: *mut u8, _layout: Layout) {}
+}
+
+#[global_allocator]
+static ALLOC: DummyAllocator = DummyAllocator;
+
+#[alloc_error_handler]
```

Comment:
What's the reason for bringing in alloc?

Reply:
`panic_unwind` needs that.

Reply:
Is there a reason we're using the `panic_unwind` feature and not `#[panic_handler]` with an `eh_personality` item? This would remove the need for the `panic_unwind` and `alloc_error_handler` features.  I believe that's the more common and less verbose way to handle panics in a custom way.

Reply:
I'm only testing a `no_std` crate with `panic_unwind` feature. Do you mean that `panic_unwind` doesn't need a `panic_handler`?

Reply:
I guess I'm asking if testing the `panic_unwind` feature is the right test here. We want to test custom panic handling, and the `panic_unwind` feature is one way to do that. Another is simply adding a `panic_handler` and `eh_personality`. Perhaps we should add another test for that?

Reply:
Thoughts here?

Reply:
Do you mean adding a test with `panic=unwind` feature and a custom panic handler? It seems somehow complicated to write a robust unwind panic handler...